README.md

```diff
@@ -26,8 +26,8 @@ Options for fmt:
 
 Options for sync:
   -e, --exercise <slug>        Only operate on this exercise
-  -o, --offline                Do not check that the cached 'problem-specifications' directory is up to date
-  -u, --update                 Prompt to update the seen data that are unsynced
+  -o, --offline                Do not update the cached 'problem-specifications' data
+  -u, --update                 Prompt to update the seen track data that are unsynced
```

Comment:
What does "seen" mean here?

```suggestion
  -u, --update                 Prompt to update the unsynced track data
```

Reply:
"prompt to update the unsynced track data" sounds like it operates on all unsynced data. The "seen" tries to convey that it respects the `--docs`, `--filepaths`, `--metadata`, and `--tests` options - otherwise a user might think those only work without `--update`.

It doesn't help that the help descriptions are like

```
--metadata               Sync Practice Exercise '.meta/config.json' metadata values
```

where it says just "Sync". In general there's a lot of potential confusion from "sync" vs "update".

Reply:
Ah. I didn't quite get that, but I must admit that I don't know a better option.

Reply:
Hmm. Thoughts on this?

```
  -u, --update                 Prompt to update unsynced track data (default: docs, filepaths, metadata, and tests)
```

This tries to convey what "seen" tried to, and it would also document the default syncing scope in the help message.

Reply:
Actually, I don't love that either. That's the default syncing scope even without update, so it should be written somewhere else.

Reply:
Agreed. What about:

> Prompt to update selected unsynced track data

But that implies a selection to be made

Reply:
Yeah. 

Let's go with

> "Prompt to update the unsynced track data"

for now, and try to improve later? Maybe nobody notices the problems here as much as us.

Reply:
👍  The latter is almost certainly true :)